pipes.ts

```diff
@@ -36,8 +36,8 @@ namespace jacdac {
         }
 
         openCommand(cmd: number) {
-            const b = Buffer.pack("IIHH", [0, 0, this.port, 0])
-            b.write(0, Buffer.fromHex(selfDevice().deviceId))
+            const b = jdpack<[Buffer, number, number]>("b[8] u16 u16",
```

Comment:
should we have a device id data type?

Reply:
deviceid is semantics - eg we don't have accelerometer force type; I would stay with `b[8]`